### DISCUSSION TITLE
Get artist count on album?

### DISCUSSION THREAD

**Discussion by RollingStar** [beetbox/beets#3921] [General]
Is beets aware of the number of artists in an album? This OST has 5 artists which sure feels like "Various Artists" to me, even though they're on every track. On my collection I might want a rule like >=4 artists is "Various Artists".

looks like the API returns individual artists which are then joined:

https://musicbrainz.org/ws/2/release/1349fb6c-784b-4ebf-b873-28fbf3a7cca8?inc=artist-credits+labels+discids+recordings&fmt=json

**Comment by sampsyo** [comment]
FWIW, when tagging from MusicBrainz, beet uses MB's own designation about whether a release should be considered "various artists." MB uses a special "artist ID" to indicate these, which you can see in beets here:
https://github.com/beetbox/beets/blob/e1a901dce5c7b867a5c84a438044c4093bcbed05/beets/autotag/mb.py#L32
